spec/luacov.stats.out

```diff
@@ -1,54 +0,0 @@
-163:/home/john/.luarocks/share/lua/5.1/busted/block.lua
```

Comment:
I think this file should not be here -- you have stats.out in the .gitignore.

Reply:
It's already deleted, see: https://github.com/kleisauke/lua-vips/tree/cleanup/spec